deploy/kubernetes/examples/demo-storageclass-config-secret.yaml

```diff
@@ -0,0 +1,20 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+volumeBindingMode: WaitForFirstConsumer  # Optional. Required for Topology Awareness.
+parameters:
+  # non secret related parameters can be provided in by_system_id parameter for each system or by parameter name as default.
+
+  by_system_id: '{"demo-system-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"thin","volume_name_prefix":"demo-prefix-1"},
+                  "demo-system-id-2":{"pool":"demo-pool-2","volume_name_prefix":"demo-prefix-2"}}'  # Optional.
+  pool: demo-pool
+  SpaceEfficiency: thin  # Optional.
+
+  csi.storage.k8s.io/secret-name: demo-config-secret
+  csi.storage.k8s.io/secret-namespace: default
+
+  csi.storage.k8s.io/fstype: xfs   # Optional. Values ext4\xfs. The default is ext4.
+  volume_name_prefix: demo-prefix  # Optional.

```

Comment:
while we tidy things up here, could we also move the sidecar parameters to the bottom? so that the parameters are grouped by their recipient

Reply:
done

Reply:
should fstype come before the secret parameters (lexicographically and visually for the `# Optional.` comments)?

Reply:
ok done